ClearentIdtechIOSFramework/ClearentUI/ClearentProcessingModalViewController.swift

```diff
@@ -133,6 +128,17 @@ extension ClearentProcessingModalViewController: ClearentProcessingModalView {
 
     // MARK: - Private
     
+    private func shouldDisplayOfflineModeLabel() -> Bool {
+        if ClearentWrapper.configuration.enableOfflineMode {
+            let offlineModeConfirmedDuringPayment = [.payment].contains(ClearentWrapper.shared.flowType?.processType) && presenter?.isOfflineModeConfirmed == true
```

Comment:
can this be written like this?
```suggestion
            let offlineModeConfirmedDuringPayment = .payment == ClearentWrapper.shared.flowType?.processType && presenter?.isOfflineModeConfirmed
```